indico/modules/rb_new/__init__.py

```diff
@@ -40,6 +43,27 @@ def _sidemenu_items(sender, event, **kwargs):
                            icon='location')
 
 
+@signals.users.merged.connect
+def _merge_users(target, source, **kwargs):
+    BlockingPrincipal.merge_users(target, source, 'blocking')
+    Blocking.find(created_by_id=source.id).update({Blocking.created_by_id: target.id})
+    Reservation.find(created_by_id=source.id).update({Reservation.created_by_id: target.id})
+    Reservation.find(booked_for_id=source.id).update({Reservation.booked_for_id: target.id})
+    Room.find(owner_id=source.id).update({Room.owner_id: target.id})
+    rb_settings.acls.merge_users(target, source)
```

Comment:
we also need this one (forgot that when adding the acls):

    RoomPrincipal.merge_users(target, source, 'room')